test/e2e/network/dns.go

```diff
@@ -597,7 +599,38 @@ var _ = common.SIGDescribe("DNS", func() {
 		}
 		validateDNSResults(ctx, f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
+})
+
+var _ = common.SIGDescribe(feature.RelaxedDNSSearchValidation, framework.WithFeatureGate(features.RelaxedDNSSearchValidation), func() {
```

Comment:
we can try to add a TODO and try to label directly
```suggestion
// TODO replace WithLabel by framework.WithFeatureGate(features.RelaxedDNSSearchValidation) once https://github.com/kubernetes/kubernetes/pull/126977 is solved
var _ = common.SIGDescribe(feature.RelaxedDNSSearchValidation, framework.WithLabel(f"Feature:Alpha"), func() {
```